# Fix to allow new okta format.

--- a/gp-okta.py
+++ b/gp-okta.py
@@ -361,8 +361,8 @@
 
 def get_redirect_url(conf, c, current_url=None):
 	# type: (Conf, str, Optional[str]) -> Optional[str]
-	rx_base_url = re.search(r'var\s*baseUrl\s*=\s*\'([^\']+)\'', c)
-	rx_from_uri = re.search(r'var\s*fromUri\s*=\s*\'([^\']+)\'', c)
+	rx_base_url = re.search(r'\bbaseUrl\b[ "]*[=:][ "\']*([^\'"]+)', c)
+	rx_from_uri = re.search(r'\bfromUri\b[ "]*[=:][ "\']*([^\'"]+)', c)
 	if not rx_from_uri:
 		dbg(conf.debug, 'not found', 'formUri')
 		return None
